Network/network-display.5s.sh

```diff
@@ -0,0 +1,72 @@
+
+SYSTEM_GLOBAL_IPV4=$(echo "show State:/Network/Global/IPv4" | scutil)
+PRIMARY_SERVICE_GUID=$(echo "${SYSTEM_GLOBAL_IPV4}" | grep "PrimaryService" | awk '{print $3}')
+PRIMARY_DEVICE_NAME=$(echo "${SYSTEM_GLOBAL_IPV4}" | grep "PrimaryInterface" | awk '{print $3}')
+if [ ! -z "${PRIMARY_SERVICE_GUID}" ]; then
+    SERVICE_INTERFACE=$(echo "show Setup:/Network/Service/${PRIMARY_SERVICE_GUID}/Interface" | scutil)
+    HARDWARE_TYPE=$(echo "${SERVICE_INTERFACE}" | grep "Hardware :" | awk -F': ' '{print $2}')
+    SERVICE_NAME=$(echo "${SERVICE_INTERFACE}" | grep "UserDefinedName :" | awk -F': ' '{print $2}')
+
+    if [ "${HARDWARE_TYPE}" == "AirPort" ]; then
+        WIFI_CURRENT_SSID=$(networksetup -getairportnetwork "${PRIMARY_DEVICE_NAME}" | awk -F': ' '{print $2}')
+
+        echo "${WIFI_CURRENT_SSID} | size=12 templateImage='${WIFI_ICON}'"
+    elif [ "${HARDWARE_TYPE}" == "Ethernet" ]; then
+        echo "| templateImage='${ETHERNET_ICON}'"
+    else
+        echo "♒︎"
+    fi
+    echo "---"
+    echo "Primary | size=12"
+    echo "${SERVICE_NAME:-Unknown} (${PRIMARY_DEVICE_NAME})"
+    echo "---"
+else
+    echo "❌"
+    echo "---"
+fi
+
+echo "Network Service | size=12"
+SERVICE_LIST=$(networksetup -listnetworkserviceorder)
+IFS=$'\n'
+for SERVICE_NAME_ST in $(echo "${SERVICE_LIST}" | grep '^([0-9*])'); do
+    SERVICE_NAME=$(echo "${SERVICE_NAME_ST}" | sed "s/([0-9*]) \(.*\)/\1/")
+    SERVICE_STATE=$(networksetup -getnetworkserviceenabled "${SERVICE_NAME}")
+
+    DEVICE_NAME=$(echo "${SERVICE_LIST}" | grep "Hardware Port: ${SERVICE_NAME}" | awk -F'Device: ' '{print $2}' | cut -d')' -f1)
+    IFCONFIG_OUT=$(ifconfig ${DEVICE_NAME} 2>/dev/null)
```

Comment:
```
In /Users/travis/build/matryer/bitbar-plugins/Network/network-display.5s.sh line 60:
    IFCONFIG_OUT=$(ifconfig ${DEVICE_NAME} 2>/dev/null)
                            ^-- SC2086: Double quote to prevent globbing and word splitting.
```